app/src/main/java/io/neurolab/activities/ProgramModeActivity.java

```diff
@@ -95,6 +95,8 @@ public void onBackPressed() {
         super.onBackPressed();
         if (mode == FOCUS_PROGRAM_MODE)
             startActivity(new Intent(this, FocusParentActivity.class));
+        if(mode == RELAX_PROGRAM_MODE)
+            startActivity(new Intent(this,RelaxParentActivity.class));
```

Comment:
Please reformat your code at all places. Use Android studio features of rearrange and reformat for this.